### PR TITLE
Product and Community docs

### PR DESCRIPTION
**Pull request by choldgraf**
This ports our Product and Community Lead google doc + blog post into our team compass, as the starting point for a set of expectations, job description, etc for this role. I believe we are still figuring out the details here, but I think this is a good starting point to iterate from, and helps set public expectations of what the person who is hired can expect.

Once somebody begins this role, I think we should re-visit this page with them and understand how it matches / mis-matches their vision for it, and potentially make some updates and tweaks accordingly.

### CODE REVIEW

File: hr/titles/product-community-lead.md [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+We are still developing the responsibilities and expectations for the role, and will revisit this as we've come to a better understanding of them.
+
+Here is a short list of relevant ideas. Note that this role may do many of these as an "individual contributor" inititally due to our small size, but ultimately the goal is for this role to grow into a leadership and strategy position that may guide and manage others.
+
+- Actively reach out to communities that we serve and learn about their goals and how 2i2c's infrastructure fits into those goals.
+- Actively guide our engineering team's efforts around design and prioritization.
+- Help us consider and discuss trade-offs that we must make in support requests and features.
+- Design and grow structures, spaces, etc for communication between 2i2c and communities we serve, as well as within communities.
+- Lead efforts to provide support for our communities in their use of the infrastructure, and route requests to the engineering team as needed.
+- Lead content creation to empower communities to use our infrastructure more effectively
```
Comment (damianavila) [inline hr/titles/product-community-lead.md:46]:
```suggestion
- Lead content creation to empower communities to use our infrastructure more effectively.
```

File: hr/titles/product-community-lead.md [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+The skills, interests, and career paths for interfacing with communities, guiding them, and participating in design and prioritization efforts with engineers are significantly different from engineering track jobs.
+As such, this role will lead efforts around a new _class_ of jobs within 2i2c: Product and Community.
+
+## Qualities and experience
+
+We are still defining the qualities and experience for this role, but below are a few abstract qualities that will make somebody succeed in this role.
+
+- Demonstrated ability to lead and grow distributed communities of practice (e.g., 3+ years as a leader of an open source community)
+- Demonstrated ability to design and manage technical products (e.g., 3+ years experience as a product manager, or as a product-level lead in an open source community)
+- Demonstrated experience in growing and guiding organizations and teams that do technical development or service work (e.g., 3+ years as a manager or organizational leadership role)
```
Comment (damianavila) [inline hr/titles/product-community-lead.md:32]:
```suggestion
- Demonstrated experience in growing and guiding organizations and teams that do technical development or service work (e.g., 3+ years as a manager or organizational leadership role).
```

File: hr/titles/product-community-lead.md [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+- Define strategy, goals, and assessment criteria for Product and Support.
+- Define roles and job titles for team members that do work in this area, as well as their connections with others on the team.
+- As resources are available, manage and mentor other team members within Product and Community at 2i2c.
+- Interface with the Partnerships Lead to incorporate sustainability in the design and evolution of our services.
+
+## Salary rationale
+
+This is a leadership-level role within 2i2c and focused on Product and Community work. As such we should use the GuideStar salary criteria for the “top” position in a relevant category. It should be appropriate for an early-mid career leader who has demonstrated some experience in the organizational aspects needed to succeed in this role.
+
+We'll plan to use the **Top Program Position** job category within GuideStar to define the base salary for this role. The median base salary for this role is $122,000.
```
Comment (damianavila) [inline hr/titles/product-community-lead.md:56]:
```suggestion
We'll plan to use the **Top Program Position** job category within GuideStar to define the base salary for this role. The median base salary for this role is $122,000 as of now (mid-2022).
```

**Review by damianavila — APPROVED** [review]
Minor suggestions, LGTM after those are addressed.

**Comment by choldgraf** [comment]
thanks for the suggestions! I've accepted them all so will merge now!

**Pull request merged**